src/dodal/beamlines/i22.py

```diff
@@ -65,6 +108,7 @@ def saxs(
     )
 
 
+@skip_device(lambda: BL == _LAB_NAME)
 def synchrotron(
     wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
```

Comment:
```suggestion
def synchrotron(
    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = BL == _LAB_NAME
```
simulate the device for the lab?